pkgs/applications/science/biology/nest/default.nix

```diff
@@ -37,24 +40,36 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [
     cmake
     pkg-config
-    autoPatchelfHook
   ];
 
   buildInputs = [
     gsl
     readline
     libtool # libltdl
+    boost
+    findutils
```

Comment:
Should `findutils` go to `nativeBuildInputs`?

Reply:
Yes it should thank you for pointing it out, it has now been corrected.